data/core/tips.json

```diff
@@ -342,7 +322,7 @@
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Put a funnel over a jug or barrel to collect rainwater over time." ]
+    "text": [ "Put a funnel over a jug or barrel to collect rainwater over time.  Downspouts work too." ]
```

Comment:
This is a good candidate for removal, actually. It was already not great advice before #34515 because clean water is easy to make from infinite sources like rivers and swimming pools. With #34515, the yields from funnels are microscopic.

Reply:
@Kodiologist it still works, when it rains, so advice is valid. Though, you can't use it 95% of the time.:smile:

Reply:
I do think a tip about getting water from running sources might be in order.

Reply:
A more available source are the pools in forests.  And salt water in swamps is also nice to know.